tests/Application.CommonTests/Builders/ProductBuilder.cs

```diff
@@ -1,32 +1,98 @@
 ﻿using CleanCompanyName.DDDMicroservice.Domain.Entities.Products;
-using CleanCompanyName.DDDMicroservice.Domain.Entities.Products.ValueObjects;
 
 namespace CleanCompanyName.DDDMicroservice.Application.CommonTests.Builders;
 
 public static class ProductBuilder
 {
     public static Product GetProductEmpty()
     {
-        return new Product();
+        return Product.Create
```

Comment:
You're all the time using Product.Create and I think that is conceptually wrong because you just want a new instance and the create could mean something else... (update stock, start campaign, whatever)